examples/llms/providers/customChatProvider.ts

```diff
@@ -77,6 +78,10 @@ export class CustomChatLLM extends ChatLLM<CustomChatLLMOutput, CustomGenerateOp
     return { tokenLimit: Infinity };
   }
 
+  async embed(_input: BaseMessage[][], _options?: EmbeddingOptions): Promise<number[][]> {
```

Comment:
Im a little surprised to see embed on the chat interface, is there a use case we have in mind for this?

Reply:
since it is an abstract method in BaseLLM, it needed to be implemented in all classes that extended it

Reply:
Why messages could not be embedded?

Reply:
There is no technical reason, but it seems a little odd to me i.e. Im not sure how it would be useful.